asllib/doc/ASLTypingReference.tex

```diff
@@ -4753,6 +4809,30 @@ \section{TypingRule.EGetBitFieldTyped \label{sec:TypingRule.EGetBitFieldTyped}}
 
 \isempty{\subsection{Comments}}
 
+\section{TypingRule.EGetTupleItem \label{sec:TypingRule.EGetTupleItem}}
+
+\subsection{Prose}
+All of the following apply:
+\begin{itemize}
+  \item $\ve$ denotes \texttt{e1, field\_name};
+\end{itemize}
+
+\subsection{Example}
```

Comment:
Here, and in fact everywhere: we need small representative examples like in the semantics reference, which should also be used for regression.

Reply:
absolutelu. I think that after the beta I'm going to add a lot of non-tricial examples that demonstrate the tricky parts.

Reply:
Fantastic, thank you.